### DISCUSSION TITLE
Agridder Monitor Pop Up

### DISCUSSION THREAD

**Discussion by louipop** [apohl79/audiogridder#716] [Help]
I unclick everythinh to hide the Agridder Monitor Pop Up.. in the menubar Trayicon on Catalina ..
but after restart Logic Pro X  it always appear again no matter what,,, the Agridder Monitor Pop Ups again
is there a way to disable it?...  completely?

**Comment by apohl79** [comment]
You can disable the tray app in 1.2 beta 5.

**Comment by louipop** [comment]
thanks ¡ but now in 1.2 Beta if I disable it.. everytime i try to load a plugin Logic Pro X + Catalina + Parallels crashes.. every time.. i have to enable it so it does not crashes Logic ..

**Reply by apohl79** [reply]
I think I see that bug. I missed the recents mechanism, which is using the tray app. Creating a bug ticket.
